ocs_ci/ocs/scale_lib.py

```diff
@@ -80,150 +77,151 @@ def create_and_set_namespace(self):
         else:
             self.sa_name = None
 
-    def create_multi_pvc_pod(
-        self, pods_per_iter=5, io_runtime=3600, start_io=False, pvc_size=None
-    ):
+    def create_multi_pvc_pod(self, pvc_count=760, pvcs_per_pod=20, obj_name="obj1"):
         """
         Function to create PVC of different type and attach them to PODs and start IO.
 
         Args:
-            pods_per_iter (int): Number of PVC-POD to be created per PVC type
-            Example, If 2 then 8 PVC+POD will be created with 2 each of 4 PVC types
-            io_runtime (sec): Fio run time in seconds
-            start_io (bool): If True start IO else don't
-            pvc_size (Gi): size of PVC
+            pvc_count (int): Number of PVCs to be created
+            pvcs_per_pod (int): No of PVCs to be attached to single pod
+            Example, If 20 then a POD will be created with 20PVCs attached
+            obj_name (string): Object name prefix string
+            tmp_path (pathlib.Path): Directory where a temporary yaml file will
 
         Returns:
-            pod_objs (obj): Objs of all the PODs created
-            pvc_objs (obj): Objs of all the PVCs created
+            rbd_pvc_name (list): List all the rbd PVCs names created
+            fs_pvc_name (list): List all the fs PVCs names created
+            pod_running_list (list): List all the PODs names created
 
         """
-        rbd_sc = helpers.default_storage_class(constants.CEPHBLOCKPOOL)
-        cephfs_sc = helpers.default_storage_class(constants.CEPHFILESYSTEM)
-        pvc_size = pvc_size or f"{random.randrange(15, 105, 5)}Gi"
-        fio_size = get_size_based_on_cls_usage()
-        fio_rate = get_rate_based_on_cls_iops()
-        logging.info(f"Create {pods_per_iter * 4} PVCs and PODs")
-        # Create PVCs
-        cephfs_pvcs = helpers.create_multiple_pvc_parallel(
-            sc_obj=cephfs_sc,
+
+        # Condition to check kube_job batch count, value more than 750 per job
+        # will lead to failure in kube_job completion, below value is 1200
+        # since it will be divided by 2 i.e. 600 per job max as per below condition
+        if pvc_count > 1200:
```

Comment:
Should we break the jobs into more than one if pvc count is more than 1

Reply:
I am actually doing it in create_scale_pods() func line 205, which does breaks, if 1500 PVC creations then iterates 2 times to reach 1500 pods i.e, 750 * 2. but again this function can be called directly just for creations so added this check.